eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -2634,4 +2634,20 @@
     <Method name="propertyOnlyFlattenSerialize"/>
     <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
   </Match>
+
+  <!-- Exclude BC_UNCONFIRMED_CAST_OF_RETURN_VALUE as false positive -->
```

Comment:
This seems to be a spotbugs issue. Adding a tracking item https://github.com/Azure/azure-sdk-for-java/issues/28766